docsite/rst/playbooks_tags.rst

```diff
@@ -29,20 +29,42 @@ On the other hand, if you want to run a playbook *without* certain tasks, you co
 
     ansible-playbook example.yml --skip-tags "notification"
 
+
+.. _tag_inheritance:
+
+Tag Inheritance
+```````````````
+
+You can apply tags to more than tasks, but they ONLY affect the tasks themselves, everywhere else is just a
+convenience so you don't have to write it on every task::
```

Comment:
I'd split these sentences up. Maybe:  You can apply tags to more than tasks, but they *ONLY* affect the tasks themselves.  Applying tags elsewhere is just a convenience, so that you don't have to write it on every task::